src/services/types.ts

```diff
@@ -1236,6 +1238,7 @@ export interface ReferencedSymbolEntry extends ReferenceEntry {
     isDefinition?: boolean;
 }
 
+/** @internal */

```

Comment:
Need to revert this one; it's used :( https://github.com/search?q=%2Fts%5C.SymbolDisplayPartKind%2F&type=code